bin/leanpkg.bat

```diff
@@ -0,0 +1,8 @@
+@ECHO OFF
+
+SET LEANDIR=%~dp0%../
+SET LIBDIR=%LEANDIR%\lib\lean

```

Comment:
This does not work when running `leanpkg.bat` directly from the git checkout, right?  The shell scripts all work in the git-checkout as well (without installation).